packaging/PLATFORM_SUPPORT.md

```diff
@@ -0,0 +1,171 @@
+<!--
+title: "Netdata platform support policy"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/PLATFORM_SUPPORT.md
+-->
+
+# Netdata platform support policy
+
+Netdata defines three tiers of official support: 
+- [Core](#core)
+- [Intermediate](#intermediate)
+- [Community](#community)
+Each tier defines different guarantees for platforms in that tier, described below in the section about that tier.
+
+Additionally, we define two categories for special cases that we do not support:
+- third-party supported platforms
+- previously supported platforms. 
+These two categories are explained further below.
+
+Any platforms not listed in any of these categories may or may not work.
```

Comment:
broken formatting

<img width="1214" alt="Screenshot 2022-01-12 at 21 46 25" src="https://user-images.githubusercontent.com/22274335/149203094-7d4ac31f-89a3-4a7b-be70-56d1c9dcec13.png">

I think we need an empty line after at the end of each list. Also, adding an empty line before a list is a good practice too.

Reply:
Thanks for pointing it out. I fixed it in another review.